components/homme/src/share/schedule_mod.F90

```diff
@@ -207,7 +207,7 @@ subroutine genEdgeSched(elem, PartNumber,LSchedule,MetaVertex)
        endif
     enddo
     if(iam == 1) then 
-       print *,'getMetaSchedule: tmpP: ',tmpP
+       write(iulog, *),'getMetaSchedule: tmpP: ',tmpP
```

Comment:
Crusher's crayclang is reporting a typo here for `SMS_D_Ln5.conusx4v1_r05_oECv3.F2010.crusher_crayclang`:
```
ftn-1725 ftn: ERROR GENEDGESCHED, File = ../../../gpfs/alpine/cli133/proj-shared/testing/E3SM/components/homme/src/share/schedule_mod.F90, Line = 210, Column = 23
  Unexpected syntax while parsing the WRITE statement : "operand" was expected but found ",".

Cray Fortran : Version 14.0.0 (20220420160222_c98838affc7b58fed2a72f164d77c35e1bc8772f)
Cray Fortran : Compile time:  0.0158 seconds
Cray Fortran : 470 source lines
Cray Fortran : 2 errors, 0 warnings, 0 other messages, 0 ansi
```

Reply:
Ah, yeah, I've seen that error with very pedantic compilers in the past. I can remove the extra comma in a second Homme PR i'm working on.